packages/xstate-wallet/src/store/memory-store.ts

```diff
@@ -172,21 +183,24 @@ export class MemoryStore implements Store {
   }
 
   public async getLedger(peerId: string) {
-    const ledgerId = this._ledgers[peerId];
+    const ledgerId = await this.backend.getLedger(peerId);
 
     if (!ledgerId) throw new Error(`No ledger exists with peer ${peerId}`);
 
     return await this.getEntry(ledgerId);
   }
 
   public setLedger(entry: MemoryChannelStoreEntry) {
+    // !! SP: this function is not async!.... so that saved me!
```

Comment:
This is not a store function. The virtual funding workflow requires a store to be set up a certain way, and that way has not been figured out yet because budgeting is still not done.

Reply:
All functions that needed so, have been made **async** for the moment.